src/app/children/child-details/child-details.component.html

```diff
@@ -55,8 +56,14 @@ <h1 class="page-header section-child">
 
 
       <div fxLayout='row' fxLayout.xs='column wrap' fxLayout.md='column wrap' fxLayout.sm='column wrap'>
+
         <div fxFlex='160px'>
-          <img [src]="child.getPhoto()" class="child-pic" alt="child's photo">
+          <img *ngIf="!creatingNew && !editing" [src]="image" class="child-pic" alt="child's photo">
+
+          <input style="display: none" type="file" (change)=uploadChildPhoto($event) #fileUpload>
+          <button *ngIf="creatingNew || editing" class="child-pic-upload" (click)="fileUpload.click()">
+            <mat-icon class="upload-icon" fontIcon="fa-plus-circle"></mat-icon>
+          </button>
           <br>
           <mat-form-field *ngIf="creatingNew || editing" class='photo-filename'>
             <input matInput formControlName="photoFile" placeholder="Photo File Name" title="photoFile" type="text" >
```

Comment:
once the upload is working and tested, we can remove the input field for the file name again